### DISCUSSION TITLE
Add "Read/write charge periods" support for H3 models

### DISCUSSION THREAD

**Discussion by cawolf** [nathanmarlor/foxess_modbus#465] [Ideas]
Hi, I'm enjoying your plugin, thank you for all the good work! 

I was wondering if there any technical reasons, that the charge periods feature is not available for H3 models. I would really like to set them dynamically depending on the solar forecast (e.g. disabling it for sunny days). Is this possible to implement?

**Comment by canton7** [comment]
The H3s simply do not support setting charge intervals (or any other settings) unfortunately.

**Reply by cawolf** [reply]
Thank you for the quick reply, then I will look for a workaround.
